homeassistant/components/sensor/vasttrafik.py

```diff
@@ -116,7 +117,7 @@ def device_state_attributes(self):
     def state(self):
         """Return the next departure time."""
         if not self._departureboard:
-            _LOGGER.warning(
+            _LOGGER.debug(
```

Comment:
We shouldn't have side effects like logging in state properties at all. Please move this to `update`.